storage/blob_test.go

```diff
@@ -168,7 +168,7 @@ func (s *StorageBlobSuite) TestContainerExists(c *chk.C) {
 	c.Assert(ok, chk.Equals, true)
 }
 
-func (s *StorageBlobSuite) TestCreateDeleteContainer(c *chk.C) {
+func (s *StorageBlobSuite) TestCreateContainerDeleteContainer(c *chk.C) {
```

Comment:
I ctrl-f'ed TestCreateContainer and wondered how come I missed this one, turns out it was just naming thing. Better searchability fix.